ddsrouter_core/test/blackbox/ddsrouter_core/dds/local/DDSTestLocal.cpp

```diff
@@ -81,8 +81,8 @@ DdsRouterConfiguration dds_test_simple_configuration(
         topic_keyed.type_name = "HelloWorldKeyed";
         topic_keyed.topic_qos.keyed = true;
 
-        conf.builtin_topics.insert(utils::Heritable<core::types::DdsTopic>::make_heritable(topic));
-        conf.builtin_topics.insert(utils::Heritable<core::types::DdsTopic>::make_heritable(topic_keyed));
+        conf.ddspipe_configuration.builtin_topics.insert(utils::Heritable<core::types::DdsTopic>::make_heritable(topic));
+        conf.ddspipe_configuration.builtin_topics.insert(utils::Heritable<core::types::DdsTopic>::make_heritable(topic_keyed));
```

Comment:
Also consider setting and testing manual_topics.

Reply:
The manual topics are not testable at this level. See https://github.com/eProsima/DDS-Pipe/pull/64#discussion_r1365634825.